src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100000.mdx

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
I added one more PR into this release since it's a bug fix for a customer so we will want to add that into the release notes as well. Here's the PR: https://github.com/newrelic/newrelic-python-agent/pull/1222